### ISSUE TITLE
Breaking API change in version 8.28.0

### ISSUE THREAD

**Issue by iclanton** [thebuilder/react-intersection-observer#381]
The `default` export was removed in version `8.28.0`, so projects that import from `react-intersection-observer` with a statement like `import Observer from 'react-intersection-observer';`, and depend on the package with a carat (i.e. - `^8.0.0`) break when version `8.28.1` is installed.

This (https://github.com/thebuilder/react-intersection-observer/commit/1537592aa2a31d5a6c3110051ebeb62a5642d39f#diff-38682f47b0cf12c516bc3714fc45d0fdL6) appears to be the changed line that causes this break.

**Comment by thebuilder** [comment]
Yes, sorry about that - I failed at making a proper automated semantic release, since i actually wanted this to have been released as version `9.0.0`. I had counted on the actual usage of this being low, since it was deprecated in version 8.x, and remove from the documentation. It is noted in the [release changelog](https://github.com/thebuilder/react-intersection-observer/releases/tag/v8.28.0). 

It should be an easy fix (`import { InView } from 'react-intersection-observer';`)
